go-controller/vendor/github.com/coreos/go-iptables/iptables/iptables.go

```diff
@@ -45,14 +45,21 @@ func (e *Error) Error() string {
 	return fmt.Sprintf("running %v: exit status %v: %v", e.cmd.Args, e.ExitStatus(), e.msg)
 }
 
+var isNotExistPatterns = []string{
+	"Bad rule (does a matching rule exist in that chain?).\n",
+	"No chain/target/match by that name.\n",
+	"No such file or directory",
```

Comment:
hmm wait chain does not exist has a no such file or directory match error?

Reply:
I didnt add this, this just came from version bump

Reply:
ahhhh I could have saved myself some trouble if only I had checked what your changes were versus the auto changes, haha :) and +1 to your hack changes being a separate commit :D over the bump

Reply:
https://github.com/coreos/go-iptables/commit/077e672b3808a90d5be13111c722a73f50479153#r140932737 is the only thing but otherwise your changes are lgtm. I should have checked your repo instead of checking the whole commit, silly me.

Reply:
responded there. I think that code is OK and we dont need to change the vendor stuff.